pkg/hosts/hosts.go

```diff
@@ -173,17 +173,6 @@ func (h *Hosts) Remove(hosts []string) error {
 		return err
 	}
 
-	uniqueHosts := map[string]bool{}
-	for i := 0; i < len(hosts); i++ {
-		uniqueHosts[hosts[i]] = true
-	}
-
-	var hostEntries = make(map[string]struct{}, len(uniqueHosts))
-
-	for key := range uniqueHosts {
-		hostEntries[key] = struct{}{}
-	}
-
```

Comment:
Does this removal part fix the issue or it was just dead code?

Reply:
it is part of fix, as it become dead code, after fix